mlflow/spark.py

```diff
@@ -627,7 +628,8 @@ def predict(self, pandas_df):
 
 
 @experimental
-def autolog():
+@autologging_integration(FLAVOR_NAME)
+def autolog(disable=False):  # pylint: disable=unused-argument
     """
     Enables automatic logging of Spark datasource paths, versions (if applicable), and formats
```

Comment:
For consistency with docstring updates in #3815, can we update this to say:

```suggestion
    Enables (or disables) and configures logging of Spark datasource paths, versions (if applicable), and formats
```